lib/inspec/objects/attribute.rb

```diff
@@ -89,6 +89,9 @@ def to_s
     private
 
     def validate_required(value)
+      # skip if we are not doing a exec (archive/vendor)
```

Comment:
Should be `an exec`, I also think the below is easier to understand but I'll leave that to you:

```
# Skip if we are using `inspec exec` (`inspec archive/vendor` is okay)

Reply:
Whoops...I got this backwards. You've got it right.